src/Entity/User.php

```diff
@@ -279,6 +295,8 @@ public function updatedTimestamps(): void
         $this->setUpdatedAt(new DateTime('now'));
```

Comment:
`now` is the default value for date time: `new DateTime('now') === new DateTime()`